### PR TITLE
Using Serilog for frontend logging.

### PR DESCRIPTION
**Pull request by karok2m**

### CODE REVIEW

File: samples/src/Microsoft.Azure.IIoT.App/src/Program.cs [hunk 1]
```diff
@@ -31,7 +51,8 @@ public class Program {
                     .UseContentRoot(Directory.GetCurrentDirectory())
                     .UseStartup<Startup>()
                     .UseKestrel(o => o.AddServerHeader = false)
-                    .UseIISIntegration());
+                    .UseIISIntegration())
+                .UseSerilog();
```
Comment (monikavar) [inline samples/src/Microsoft.Azure.IIoT.App/src/Program.cs]: Indentation.

File: samples/src/Microsoft.Azure.IIoT.App/src/Startup.cs [hunk 2]
```diff
@@ -76,8 +86,7 @@ public class Startup {
         /// <param name="app"></param>
         /// <param name="env"></param>
```
Comment (dacolgit) [inline samples/src/Microsoft.Azure.IIoT.App/src/Startup.cs]: remove env parameter in comment

Comment (karok2m) [reply samples/src/Microsoft.Azure.IIoT.App/src/Startup.cs]: Done.

File: samples/src/Microsoft.Azure.IIoT.App/src/Program.cs (hunk 1 above)
Comment (karok2m) [reply samples/src/Microsoft.Azure.IIoT.App/src/Program.cs]: This is according to `Serilog.AspNetCore` [instructions](https://github.com/serilog/serilog-aspnetcore#instructions).

**Pull request merged**